server/routerlicious/packages/services-core/src/mongoCheckpointRepository.ts

```diff
@@ -62,6 +62,22 @@ export class MongoCheckpointRepository implements ICheckpointRepository {
 		documentId: string,
 		tenantId: string,
 	): { _id: string; documentId: string } & any {
+		const isError = !documentId || !tenantId ? true : false;
```

Comment:
I think this line can be simplified as `const isError = !documentId || !tenantId`?

Or you can just do `if ( !documentId || !tenantId)` ?

Reply:
You are right, I've simplified this